lib/crypto.js

```diff
@@ -79,23 +79,69 @@ const {
 } = require('internal/crypto/util');
 const Certificate = require('internal/crypto/certificate');
 
+// These function are needed because Hash() calls new Hash()
```

Comment:
Minor nits:

s/function/helper functions/

Also, might use more generic wording like 'These helper functions are needed because the constructors can use `new`, in which case V8 cannot inline the recursive constructor call' or something along those lines.